src/qibolab/instruments/zhinst.py

```diff
@@ -999,21 +999,17 @@ def measure_relax(self, exp, qubits, couplers, relaxation_time, acquisition_type
         """Qubit readout pulse, data acquisition and qubit relaxation."""
         readout_schedule = defaultdict(list)
         qubit_readout_schedule = defaultdict(list)
-        iq_angle_readout_schedule = defaultdict(list)
```

Comment:
Ideally I would like to remove this `iq_angle` stuff and just play with kernels, the problem is that I couldn't test them side by side due to the lack og hardware.

Reply:
I removed this variable because it was unnecessary - the same thing is anyways available in the qubit objects. I largely didn't touch how the measurements are added to the experiment. That I left for future. I believe introduction of logical channels will affect this a lot, so we can re-consider the implementation at that point